### PR TITLE
Remove `Any` from API

### PR DESCRIPTION
**Pull request by dmdashenkov**
In this PR we remove the need of an API user to convert values to `google.protobuf.Any`. Previously, `Any`s where expected as ID values when building a query to the server. Now, we accept `Object` IDs and convert them to `Any` or throw an exception if we can't.

### CODE REVIEW

**Comment by dmdashenkov** [comment]
@armiol, PTAL.

File: client/lib/target_builder.dart [hunk 1]
```diff
@@ -33,17 +36,33 @@ Target targetAll(GeneratedMessage instance) {
...
     var filters = TargetFilters();
     var idFilter = IdFilter();
-    idFilter.id.addAll(ids);
+    idFilter.id.addAll(ids.map(_idToAny));
     filters.idFilter = idFilter;
     target.filters = filters;
     return target;
 }
 
+Any _idToAny(Object rawId) {
```
Comment (armiol) [inline client/lib/target_builder.dart]: Won't we have something like `AnyPacker` and `Id`-packer?

It's just that a `target_builder` does not seem to be an appropriate place for such a common logic.

**Review by armiol — COMMENTED** [review]
@dmdashenkov please see my comment.

**Comment by dmdashenkov** [comment]
@armiol, PTAL again.

**Review by armiol — APPROVED** [review]

**Pull request merged**